kibana-reports/server/model/index.ts

```diff
@@ -30,6 +30,7 @@ export const dataReportSchema = schema.object({
   time_duration: schema.string(),
   //TODO: future support schema.literal('xlsx')
   report_format: schema.oneOf([schema.literal(FORMAT.csv)]),
+  limit: schema.number({ defaultValue: 10000 }),
```

Comment:
Do we want user to provide this field with value? If not, may you should use `limit: schema.maybe(schema.number({defaultValue: 1000 }))`. Then in the input validation, this field with default value will be added once it passed validation.

Reply:
I don't think we have any UI for the user controlling the field limit

Reply:
I missed that. Will add. Thanks!